nettacker/core/lib/http.py

```
@@ -90,6 +91,23 @@ def response_conditions_matched(sub_step, response):
             else:
                 condition_results["responsetime"] = []
+    # If the request used HTTP Basic auth, decode and expose the used credentials so
+    # modules can reference it via response_dependent['authorization_basic'] in their log field.
+    auth_header = sub_step.get("headers", {}).get("Authorization", "")
+    if isinstance(auth_header, str) and auth_header.startswith("Basic "):
+        try:
+            condition_results["authorization_basic"] = [
+                base64.b64decode(auth_header[len("Basic ") :]).decode(errors="ignore")
+            ]
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)